src/libraries/Common/tests/System/Net/Http/Http3LoopbackStream.cs

```diff
@@ -0,0 +1,330 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Buffers.Binary;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Net.Quic;
+using System.Threading.Tasks;
+
+namespace System.Net.Test.Common
+{
+
+    public sealed class Http3LoopbackStream : IDisposable
+    {
+        const int MaximumVarIntBytes = 8;
+        const long VarIntMax = 4611686018427387903L; // 2^62-1
```

Comment:
Nit: private for all these consts

Also same comment as earlier about using math for the value rather than magic numbers that require comments.

Reply:
Or expose VarIntMax and MaxVarIntBytes in VarIntHelpers.

Reply:
Oh wait, this is the LoopbackStream.